pom.xml

```diff
@@ -146,6 +146,7 @@
         <netty.version>4.1.100.Final</netty.version>
         <netty-codec-http2-version>4.1.100.Final</netty-codec-http2-version>
         <jersey-common>2.34</jersey-common>
+        <dropwizard.version>4.1.12.1</dropwizard.version>
```

Comment:
Curious any reason we pick this version? This version is from 2020: https://mvnrepository.com/artifact/io.dropwizard.metrics/metrics-core/4.1.12.1

Reply:
It's the version used by ZK 3.8 -> https://github.com/apache/zookeeper/blob/branch-3.8/pom.xml#L478